{"WARN": 13, "filename": "5_3_emty_raise_outside_except_warning_.py", "lineno": 4, "message": "The raise statement is not inside an except clause", "warnline": "  raise"}
